src/utils/export.ts

```diff
@@ -18,3 +20,93 @@ export const exportAsJSON = async (id: string, name?: string) => {
   const blob = new Blob([JSON.stringify(json)], {type: "application/json"});
   saveAs(blob, `${fileName(name ?? "scrumlr.io")}.json`);
 };
+
+export const getNoteVotes = (noteId: string, votings: VotingType[]) => votings[0].votes?.votesPerNote[noteId]?.total ?? 0;
+
+export const compareNotes = (a: {id: string; position: {rank: number}}, b: {id: string; position: {rank: number}}, votings: VotingType[]) => {
+  if (votings && getNoteVotes(a.id, votings) !== getNoteVotes(b.id, votings)) {
+    return getNoteVotes(a.id, votings) > getNoteVotes(b.id, votings) ? -1 : 1;
+  }
+  return a.position.rank > b.position.rank ? -1 : 1;
+};
+
+export const getChildNotes = (notes: NoteType[], votings: VotingType[], noteId: string) =>
+  notes.filter((n) => n.position.stack === noteId).sort((a, b) => compareNotes(a, b, votings));
+
+export const getAuthorName = (authorId: string, participants: ParticipantType[]) => participants.filter((p: ParticipantType) => p.user?.id === authorId)[0].user?.name;
+
+const {t} = i18n;
+const mdBoardHeader = (boardName: string) => `# ${boardName}\n`;
+
+const mdBoardProperties = (board: BoardType, participants: ParticipantType[]) => {
+  const header = `## ${i18n.t("MarkdownExport.properties")}\n`;
+  const boardId = `- ${i18n.t("MarkdownExport.boardId")}: ${board.id}\n`;
```

Comment:
```suggestion
  const boardId = `- [${i18n.t("MarkdownExport.boardId")}](${window.location.href.replace("/settings/export", "")})\n`;
```

The string replacement is not the best option. That's just quick and dirty to include the link without the necessity to use PUBLIC_URL

Reply:
What is wrong with using the environment variable/`PUBLIC_URL`?

Reply:
You have to manually set it and it's easy to forget about that